templates/definition/meter/sungrow-hybrid.yaml

```diff
@@ -108,12 +108,14 @@ render: |
   power:
     source: calc
     mul:
```

Comment:
Warum brauchts die ganze Rechnung jetzt überhaupt noch wenn die Leistung eh schon signed ist?

Reply:
> for compatibility with older firmware the value will be positive using the abs function.

Das geht doch ohnehin nicht aufgrund der anderen Änderungen- oder habe ich das falsch verstanden?

Reply:
> > for compatibility with older firmware the value will be positive using the abs function.
> 
> Das geht doch ohnehin nicht aufgrund der anderen Änderungen- oder habe ich das falsch verstanden?

nur am WiNet-S-Dongle geht es nicht mehr, (fraglich ob es überhaupt schon mal ging da address 13000 keine werte zurück liefert). Ansonsten sollte das ganze mit der "abs" funktion abwärtskompatibel sein.

alternative wäre folgendes möglich, aber dann nur in ein neuen Template:
```
  {{- if eq .usage "battery" }}
  power:
    source: modbus
    {{- include "modbus" . | indent 2 }}
    register:
      type: input
      address: 13021 # Battery power
      decode: int16
  energy:
```

Reply:
wie gesagt den Fehler am Native Port ist damit lösbar

Reply:
abs ist prima- dann könnte das also ohne was kaputt zu machen so rein?

Reply:
genau, deswegen "abs", das macht wenigstens nicht noch mehr kaputt mit der Firmware Chaos bei sungrow.